scripts/plugins/storage/sqlite/upgrade/72.sql

```diff
@@ -1,4 +1,4 @@
 ALTER TABLE fledge.users ADD COLUMN failed_attempts INTEGER DEFAULT 0;
-ALTER TABLE fledge.users ADD COLUMN block_until DATETIME DEFAULT (STRFTIME('%Y-%m-%d %H:%M:%f', 'NOW', 'localtime'));
+ALTER TABLE fledge.users ADD COLUMN block_until DATETIME DEFAULT "2024-01-01 10:10:10.55";
```

Comment:
```
ALTER TABLE fledge.users ADD COLUMN block_until DATETIME DEFAULT NULL;

UPDATE fledge.users SET block_until=current_timestamp;
```

Reply:
Blocked Until should be default as null.